koap/build.gradle.kts

```diff
@@ -58,3 +62,55 @@ kotlin {
         }
     }
 }
+
+// TODO: Put this in a separate file?
+fun String.runCommand(workingDir: File = file("./")): String {
+    val parts = this.split("\\s".toRegex())
+    val proc = ProcessBuilder(*parts.toTypedArray())
+            .directory(workingDir)
+            .redirectOutput(ProcessBuilder.Redirect.PIPE)
+            .redirectError(ProcessBuilder.Redirect.PIPE)
+            .start()
+    proc.waitFor(1, TimeUnit.MINUTES)
+    return proc.inputStream.bufferedReader().readText().trim()
+}
```

Comment:
I'm hoping the `execute` function still works in Kotlin DSL. It would simplify this considerably.

https://github.com/JuulLabs/juul-app-android/blob/05589f86de2aad7453768d15a4e415fabc65f367/gradle/versions.gradle

Reply:
Unfortunately `execute` is a Groovy string enhancement: http://docs.groovy-lang.org/latest/html/groovy-jdk/java/lang/String.html#execute()

I can simplify it a bit by using `project.exec` and move it into a `buildSrc` file to get it out of the way.